examples/all-clusters-app/esp32/main/CaseCommands.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/CASESessionManager.h>
+#include <app/server/Server.h>
+#include <lib/core/CHIPEncoding.h>
+#include <lib/shell/Commands.h>
+#include <lib/shell/Engine.h>
+#include <lib/shell/commands/Help.h>
+#include <lib/support/CodeUtils.h>
+#include <platform/CHIPDeviceLayer.h>
+
+using namespace chip::DeviceLayer;
+
+namespace chip {
+namespace Shell {
+
+Shell::Engine sSubShell;
+static NodeId sNodeId;
+
+static void OnConnected(void * context, OperationalDeviceProxy * deviceProxy)
+{
+    streamer_printf(streamer_get(), "Establish CASESession Success!\r\n");
+}
+Callback::Callback<OnDeviceConnected> sOnConnectedCallback(OnConnected, nullptr);
+static void OnConnectionFailure(void * context, PeerId peerId, CHIP_ERROR error)
+{
+    streamer_printf(streamer_get(), "Establish CASESession Failure!\r\n");
+}
+Callback::Callback<OnDeviceConnectionFailure> sOnConnectionFailureCallback(OnConnectionFailure, nullptr);
+
+void ConnectToNode(intptr_t arg)
+{
+    FabricInfo * fabricInfo                 = reinterpret_cast<FabricInfo *>(arg);
+    Server * server                         = &(chip::Server::GetInstance());
+    CASESessionManager * caseSessionManager = server->GetCASESessionManager();
+    if (caseSessionManager == nullptr)
+    {
+        ChipLogError(SecureChannel, "Can't get the CASESessionManager");
+        return;
+    }
+    caseSessionManager->FindOrEstablishSession(fabricInfo->GetPeerIdForNode(sNodeId), &sOnConnectedCallback,
+                                               &sOnConnectionFailureCallback);
+}
```

Comment:
#13644 is merged. So, instead can we have a single file Commands.cpp instead of `OnOffCommands.cpp` and `CaseCommands.cpp`? Later on, additional commands can be added to it as well.

Reply:
Done